src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1055,7 +1055,16 @@ def create_x509_csr(self, builder, private_key, algorithm):
         res = self._lib.X509_REQ_sign(
             x509_req, private_key._evp_pkey, evp_md
         )
-        assert res > 0
+        if res == 0:
+            errors = self._consume_errors()
+            assert errors[0][1:] in (
```

Comment:
Nope, left it in there from when I was looking for more than one error condition.